### PR TITLE
Select component

### PR DESCRIPTION
**Pull request by Meeve**

### CODE REVIEW

File: src/components/dropdown/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    getExpandedDropdown = () => {
+        let expandedDropDown = "";
+        if (this.state.isOpened) {
+            expandedDropDown = (<div className="dropDownOptionHolder" style={this.getPositionStylesForExpander()} ref={this.state.dropdownRef}>
+                {this.getWrapChildren()}
+            </div>);
+        }
+
+        return expandedDropDown;
```
Comment (Danon) [inline src/components/dropdown/Dropdown.js]: Zmienna na zapas jak gaba? :D

File: src/components/dropdown/Dropdown.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        } else {
+            // setState is a promise, so first references will be set, and then this method will trigger rerender
+            this.setState({});
+        }
+
+        return innerContainerStyles;
+    }
+
+    getExpandedDropdown = () => {
+        let expandedDropDown = "";
```
Comment (Danon) [inline src/components/dropdown/Dropdown.js]: Pusty string? Nie da się jakoś pustego komponentu?

File: src/components/dropdown/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    getStylesForEventCatcher = () => {
+        let styles = {};
+        const catcherBound = 30;
+        if (this.state.isOpened && this.state.dropdownRef.current) {
+            let dropdownSizes = this.state.dropdownRef.current.getBoundingClientRect();
+            let innerContainerStyles = this.getContainerStyle();
+            styles = {
+                width: dropdownSizes.width + (catcherBound * 2) + "px",
+                height: dropdownSizes.height + (catcherBound * 2) + "px",
+                top: -catcherBound + parseInt(innerContainerStyles.top) + "px",
```
Comment (Danon) [inline src/components/dropdown/Dropdown.js]: Is this reactive?

File: style/style.css [hunk 4]
```diff
@@ -2,6 +2,8 @@ body {
     margin: 0;
     padding: 0;
     background-color: #727272;
+    font-family: "Segoe UI";
```
Comment (Danon) [inline style/style.css]: `sans-serif` default?

File: style/style.css [hunk 5]
```diff
@@ -2,6 +2,8 @@ body {
...
+    font-size: 13px;
```
Comment (Danon) [inline style/style.css]: `em` instead of `px` maybe?

File: style/style.css [hunk 6]
```diff
@@ -41,6 +43,59 @@ body {
     background-color: green;
 }
 
+/* DROPDOWN */
+
+.dropdown {
```
Comment (Danon) [inline style/style.css]: No separate style file for component?

Comment (Meeve) [reply style/style.css]: I'm working on it. This will be handled in version 2.0

**Pull request merged**